Makefile

```diff
@@ -22,7 +22,7 @@ clean-pyc:
 	find . -name '*~' -exec rm -f {} +
 
 lint:
-	flake8 py_ecc
+	flake8 --max-line-length=100 py_ecc
```

Comment:
Instead, this can go in the tox.ini, like https://github.com/ethereum/ethereum-python-project-template/blob/03f00d3fbe73b0ea6bd46b5e5b4e9a4103ee0226/tox.ini#L17-L20

Reply:
yes @carver, then I like to go with earlier one so removing this.

Reply:
> do you mind if I ignore it from commit?

Sorry, do you mean to rebase and force-push? That's totally fine.